migration-from-4x.md

```diff
@@ -146,37 +151,58 @@ The [Inlite]({{site.url}}/docs-4x/themes/inlite/) theme is no longer supported i
   plugins: [ 'quickbars' ]
 }
  ```
-This will provide a similar but improved [distraction-free]({{site.baseurl}}/general-configuration-guide/use-tinymce-distraction-free/) experience in TinyMCE 5.0.
-
-#### Modern
-
-The [Modern]({{site.url}}/docs-4x/themes/modern/)  theme is no longer supported in TinyMCE 5.0.  The modern theme's UI library [`tinymce.ui.*`]({{site.url}}/docs-4x/api/tinymce.ui/) has also been deleted. This change may impact integrations depending upon the [level of customization](#initialization).
-
-> Note: For support related issues such as problems while migrating and a workaround, please contact [support](https://support.tiny.cloud/hc/en-us/requests/new). Alternatively, track developer preview issues on GitHub, [here](https://github.com/tinymce/tinymce/labels/5.x).
+This will provide a similar but improved [distraction-free]({{site.baseurl}}/general-configuration-guide/use-tinymce-distraction-free/) experience in TinyMCE 5.
 
+### Modern theme removed
 
-### Events
+The [Modern]({{site.url}}/docs-4x/themes/modern/) theme is not supported in TinyMCE 5.  The Modern theme's UI library [`tinymce.ui.*`]({{site.url}}/docs-4x/api/tinymce.ui/) has also been removed. This change may impact integrations depending upon the level of customization.
```

Comment:
Another one that existed before that could be misleading? As we're saying it's removed, but then saying it's not supported which would imply it still exists?

Reply:
Revised to ```has been removed from TinyMCE 5.```